topics/galaxy-interface/faqs/flatten-list-list-paired-to-list-paired.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Flatten a list of list of paired datasets into a list of paired datasets
+area: rule-builder 
+box_type: tip
+layout: hands_on
+contributors: [hexylena]
+---
+
+Sometimes you find yourself with a `list:list:paired`, i.e. a collection of collection of paired end data. This format is well supported by Galaxy, but doesn't always work with tools. This is easy to resolve with {% tool [Apply rules](__APPLY_RULES__) %}:
```

Comment:
This is weird, list:list:paired should work everywhere list:paired works. Do you have an example ?

Reply:
Nope, I'll remove that. It was presumed from the conversation with the user reporting the issue.